config/credentials.default.coffee

```diff
@@ -1,3 +1,5 @@
+secret = () -> Math.random().toString(36).slice(-16)
```

Comment:
@cihangir Ok, so let's do one thing at once - this PR fixes terraformer, the secret generation we can track in https://github.com/koding/koding/issues/8672.

I've removed the generation from this PR.